pkg/controller/volume/attachdetach/attach_detach_controller_test.go

```diff
@@ -262,6 +263,30 @@ func attachDetachRecoveryTestCase(t *testing.T, extraPods1 []*v1.Pod, extraPods2
 		i++
 	}
 
+	// Create the controller
+	adcObj, err := NewAttachDetachController(
+		fakeKubeClient,
+		informerFactory.Core().V1().Pods(),
+		informerFactory.Core().V1().Nodes(),
+		informerFactory.Core().V1().PersistentVolumeClaims(),
+		informerFactory.Core().V1().PersistentVolumes(),
+		informerFactory.Storage().V1().CSINodes(),
+		informerFactory.Storage().V1().CSIDrivers(),
+		nil, /* cloud */
+		plugins,
+		prober,
+		false,
+		1*time.Second,
+		DefaultTimerConfig,
+		nil, /* filteredDialOptions */
```

Comment:
I think you need to add this arg at line 177 too? How come you need to create a new controller here?

Reply:
Not sure what happened there. The diff is no reasonable.

Reply:
Sorry, I tried to write "fixed, the diff is now reasonable".

A new controller didn't need to be made, maybe there was a rebase problem.